openy_af4_vue_app/src/components/Fieldset.vue

```diff
@@ -19,11 +19,26 @@
           {{ counterOptions | formatPlural('1 result', '@count of results') }}
         </span>
         <span v-if="collapsible" class="icon">
-          <font-awesome-icon icon="plus-circle" />
-          <font-awesome-icon icon="minus-circle" />
+          <Icon
+            icon="material-symbols:add-circle-outline"
+            class="circle-plus"
+            width="1.2rem"
+            height="1.2rem"
```

Comment:
why not via css? @aleevas @rollki

Reply:
@podarok, here are the fixes https://github.com/YCloudYUSA/yusaopeny_activity_finder/pull/41/commits/e84be02f6220590394f9fd11fc457ac59ffb5ec6